engines/cryomni3d/font_manager.cpp

```diff
@@ -73,7 +73,7 @@ void FontManager::loadFont(Common::ReadStream &font_fl) {
 	font_fl.read(font->comment, sizeof(font->comment));
 	//debug("Comment %s", font.comment);
 
-	for (uint i = 0; i < Font::kCharactersCount; i++) {
+	for (int i = 0; i < Font::kCharactersCount; i++) {
```

Comment:
uint is good here, as it's an index. I would change kCharactersCount to be an uint as well